indicator_creation_rules/new_user_account_logging.py

```diff
@@ -25,4 +25,4 @@ def rule(event):
 
 
 def title(event):
-    return f"A new user account was created - [{event.udm('user')}]"
+    return f"A new user account was created - [{event.udm('user') or 'Username Unknown'}]"
```

Comment:
nit: standardize missing fields

```suggestion
    return f"A new user account was created - [{event.udm('user') or '<UNKNOWN_USER>'}]"
```